lib/auth0/mixins/httpproxy.rb

```diff
@@ -16,7 +16,7 @@ module HTTPProxy
       BASE_DELAY = 100
 
       # proxying requests from instance methods to HTTP class methods
-      %i(get post post_file put patch delete delete_with_body).each do |method|
+      %i(get post post_file post_form put patch delete delete_with_body).each do |method|
```

Comment:
Until now the SDK didn't have a way to send a POST request with `application/x-www-form-urlencoded`, it always assumed JSON. To support this, I added a new method type here (which ultimately ends up as a POST, similar to how `:post_file` is used).

The alternative was to add an option to the method but there was no way to do this without introducing a potential breaking change.